src/domain/synthetics/tradeHistory/useTradeHistory.ts

```diff
@@ -68,70 +70,267 @@ export function useTradeHistory(
     error,
     size: pageIndex,
     setSize: setPageIndex,
-  } = useInfiniteSwr<RawTradeAction[]>(getKey, {
+  } = useInfiniteSwr(getKey, {
     fetcher: async (key) => {
-      const [, , , , pageIndex] = key;
-      const skip = pageIndex * pageSize;
-      const first = pageSize;
+      const pageIndex = key.at(-2) as number;
```

Comment:
maybe just use `index` var instead?

Reply:
i might have misunderstood but there is no `index` variable in scope.